code/modules/admin/admin.dm

```diff
@@ -69,8 +71,7 @@
 		body += "<a href='?_src_=holder;[HrefToken()];modantagtokens=subtract;mob=[REF(M)]'>-</a> "
 		body += "<a href='?_src_=holder;[HrefToken()];modantagtokens=set;mob=[REF(M)]'>=</a> "
 		body += "<a href='?_src_=holder;[HrefToken()];modantagtokens=zero;mob=[REF(M)]'>0</a>"
-		var/metabalance = M.client.get_metabalance_db()
-		body += "<br><b>[CONFIG_GET(string/metacurrency_name)]s</b>: [metabalance] "
+		body += "<br><b>[CONFIG_GET(string/metacurrency_name)]s</b>: [M.client.metabalance_cached || M.client.get_metabalance_db()] "
```

Comment:
There's a proc to get cached metabalance iirc, use that instead

Reply:
The proc crashes if the metabalance isn't cached and is designed to only be used if you know for sure it has already been cached it seems.

Reply:
Done